_resources/seniorsemresources.md

```diff
@@ -75,18 +71,13 @@ LaTeX resources:
 - [Mangling image from pdfs](mangling-image-from-pdfs.pdf) 
 - [Our own LaTeXFormattingFAQ](LatexformatingFAQ.md)
 
-### ACM Computing Categories and General Terms
+## ACM Computing Categories and General Terms
 
 - [The ACM Computing Classification System](http://www.acm.org/about/class/ccs98-html)
 - General terms must be a subset of the 16 terms [listed here](http://www.acm.org/about/class/1998/). 
 
-### Creative Commons licensing
-
-   [ Creative Common Licenses](Common Licences.md)
```

Comment:
I assume this is a document that was planned to be added, but since it doesn't exist right now I removed the section.